### PR TITLE
The glowing forest anomaly

### PR DESCRIPTION
**Pull request by TheShuEd**
image

## About the PR
Added new anomaly to the game. Passively replaces floor tiles with astro-grass (can be removed with a crowbar).

During the pulses spawn a "friendly" flower carpet. This is the floor-layer analog of kudzu, which slows movement by 20%, and creates trees, crystals and glowing berries as it spreads.
Berries, when broken down, yield random vegetables, fruits and plants, which is a source of useful resources and chemicals. There is also a chance that living light creatures will emerge from the berry.
"friendly carpet" grows very slowly, and essentially poses no threat.

When the anomaly supercrites, it sparks a lot of "aggressive" flower carpet. This carpet spreads much faster, and has a chance to spawn living light monsters directly.

In sum, this creates a sort of "forest zone" in which a limited number of aggressive creatures appear to defend it.

## Why / Balance
more anomalies

## Technical details
cleaned up some components a bit, adjusting them to current standards. 

## Media

https://github.com/space-wizards/space-station-14/assets/96445749/4d69d25c-0738-465b-b165-e160fc838db7

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added new Floral anomaly!

### CODE REVIEW

**Comment by TheShuEd** [comment]
I suspect that the anomaly is quite simple to contain, but perhaps this is even good for balance against difficult to contain anomalies

File: Resources/Prototypes/Entities/Objects/Misc/kudzu.yml [hunk 1]
```diff
@@ -103,6 +103,59 @@
     - type: Kudzu
       spreadChance: 0.3
 
+- type: entity
+  id: KudzuFlowerFriendly
+  name: floral carpet
+  suffix: Friendly, Floral Anomaly
+  description: a colorful carpet of flowers sprawling in every direction. You're not sure whether to take it down or leave it up
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Misc/kudzu.yml:110]:
```suggestion
  description: A colorful carpet of flowers sprawling in every direction. You're not sure whether to take it down or leave it up.
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Specific/Anomaly/anomalies.yml [hunk 2]
```diff
@@ -290,6 +289,86 @@
...
+    superCriticalSpawns:
+    - KudzuFlowerAngry
+    spawns:
+    - KudzuFlowerFriendly
+
+- type: entity
+  id: AnomalyFloraBulb
+  name: glowing strange berry
+  parent: BaseStructure
+  description: It's a beautiful, glowing, strange berry. It seems to have something growing inside it...
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Specific/Anomaly/anomalies.yml:333]:
```suggestion
  name: strange glowing berry
  parent: BaseStructure
  description: It's a beautiful strange glowing berry. It seems to have something growing inside it... 
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Comment by deltanedas** [comment]
blessed astrograss + food farm

**Comment by K-Dynamic** [comment]
finally, spessmen can touch grass

**Pull request merged**